ca/ca.go

```diff
@@ -100,13 +106,47 @@ func (ca *CA) Init(config *authority.Config) (*CA, error) {
 	mux := chi.NewRouter()
 	handler := http.Handler(mux)
 
-	// Add api endpoints in / and /1.0
+	// Add regular CA api endpoints in / and /1.0
 	routerHandler := api.New(auth)
 	routerHandler.Route(mux)
+	mux.Route("/", func(r chi.Router) {
+		routerHandler.Route(r)
+	})
 	mux.Route("/1.0", func(r chi.Router) {
 		routerHandler.Route(r)
 	})
 
+	if config.ACME != nil {
+		//Add ACME api endpoints in /acme and /1.0/acme
+		dns := config.DNSNames[0]
+		u, err := url.Parse("https://" + config.Address)
+		if err != nil {
+			return nil, err
+		}
+		port := u.Port()
+		if port != "" && port != "443" {
+			dns = fmt.Sprintf("%s:%s", dns, port)
+		}
+		prefix := config.ACME.Prefix
+		acmeAuth := acme.NewAuthority(auth.GetDatabase().(nosql.DB), dns, prefix, auth)
+		acmeRouterHandler := acmeAPI.New(acmeAuth)
+		mux.Route("/"+prefix, func(r chi.Router) {
+			acmeRouterHandler.Route(r)
+		})
+		mux.Route("/1.0/"+prefix, func(r chi.Router) {
+			acmeRouterHandler.Route(r)
+		})
+	}
```

Comment:
This block will change to support different acme provisioners.

Reply:
We can hash this out when we talk in more detail about how to implement acme provisioners. If there's a 1:1 relationship between an acme provisioner and an acme account then I don't see a need to offer a different api, unless you're planning on serving different resources.